exercises/run-length-encoding/example.js

```diff
@@ -0,0 +1,13 @@
+
+export const encode = (plaintext) => {
+  const consecutiveChars = /([\w\s])\1*/g;
+  return plaintext.replace(consecutiveChars,
+      match => (match.length > 1 ? match.length + match[0] : match[0]));
+};
+
+
+export const decode = (cypher) => {
```

Comment:
hmm...it seemed terse and to the point at first...but on second thoughts, I think it probably would be clearer  if we renamed it to ```compressedText``` of  ```encodedText```. What do you think?